### ISSUE TITLE
Enable setting a default separator

### ISSUE THREAD

**Issue by rlivsey** [paddle8/ember-document-title#17]
It would be great if the default separator could be overridden instead of having to remember to do it every time the `{{title}}` helper is invoked.

The simplest way might be to define it in the service, then an app could simply extend it:

```javascript
// app/services/document-title-list.js
import TitleService from 'ember-document-title/services/document-title-list';

export default TitleService.extend({
  separator: ' > '
});
```

If this sounds reasonable I'll be able to make a pull request (or for an alternative implementation).

**Comment by tim-evans** [comment]
You only need to do it at the top level, but this does seem reasonable. I'd take a PR for this.

**Comment by rlivsey** [comment]
I tried setting it at the top level, but that only changed it there and not for subsequent levels.

That seems to be because the helper [sets a default regardless](https://github.com/paddle8/ember-document-title/blob/master/addon/helpers/title.js#L21), so it's never `null` when the service would [choose to use the previous separator](https://github.com/paddle8/ember-document-title/blob/master/addon/services/document-title-list.js#L23).

So the better fix than my suggestion might be to remove the default in the helper & have the service handle nulls again?

**Comment by tim-evans** [comment]
Yeah, that sounds better. Could you write a failing test case for the separator cascade?

**Comment by rlivsey** [comment]
Yep, no problem. Just about to head onto a conf call but I'll take a look this evening.

**Comment by tim-evans** [comment]
Thanks! :)

**Comment by rlivsey** [comment]
Changed `this.get(` to `get(this`

**Comment by tim-evans** [comment]
Published as `2.0.1`.
